bin/webpack-dev-server.js

```diff
@@ -368,7 +370,7 @@ function startDevServer(webpackOptions, options) {
     throw e;
   }
 
-  if (argv.progress) {
```

Comment:
a less impactful option would have been to set `options.progress` within the existing `if` block. keep an eye on those kinds of simple changes in the future and give them favor.

Reply:
I think I understand what you're saying, since the progress opt is only used here and its not always common to use, it would have been less impactful to set the progress opt inside of the if block so it wouldn't be assigned on every run? 

Thank you for the review! Looking forward to the v3 version :)